src/handlers/shared/start.ts

```diff
@@ -75,17 +75,17 @@ export async function start(context: Context, issue: Context["payload"]["issue"]
   }
 
   // get labels
-  const labels = issue.labels;
+  const labels = issue.labels ?? [];
   const priceLabel = labels.find((label: Label) => label.name.startsWith("Price: "));
 
   if (!priceLabel) {
     throw new Error(logger.error("No price label is set to calculate the duration", { issueNumber: issue.number }).logMessage.raw);
```

Comment:
For some reason this PR is not posting a comment but my PR I just opened fresh from `development` does post a comment no problem

https://github.com/ubq-testing/command-start-stop/issues/19